app/react-components/components/DownloadVisualizationButton.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import DropDownButton from './DropDownButton';
+import downloadSvg from '../utils/download-svg';
+import saveFile from '../utils/filesaver';
+import toTsvString, {mapArrayToTsvString} from '../utils/toTsvString';
+
+function getSelector(el) {
+  return typeof el === 'string' ? document.querySelector(el) : el;
+}
+const styles = {
+  hidden: {
+    width: 0,
+    height: 0,
+    overflow: 'hidden',
+  },
+};
+
+const icon = <i className={"fa fa-download"}/>;
```

Comment:
In subui we have an `/icons` folder.. we can add one here then do something like

```
let WhateverIcon = ({ classname, ...props }) => 
  <i className={`${className} fa fa-whatever} {...props} />
```

or yeah just do it inline. the advantage of the named component is in case we decided to switch what `whatever` icon should be, only to need to change it one place.

but yes please no react-icons for now. I have a long running branch in subui to remove them all.